services/graphql-server/src/graphql/resolvers/platform/content.js

```diff
@@ -482,7 +482,10 @@ module.exports = {
       if (section) return section;
 
       // Current section does not match site, load alternate.
-      // @todo This should eventually account for secondary sites/sections. For now, load home.
+      // @todo This should eventually account for secondary sites/sections.
+      // For now load an alternate from schedules.
+      const sectionFromSched = await getSectionFromSchedules({ content, siteId, load });
```

Comment:
For clarity, might as well name the variable the full name ... `sectionFromSchedule`